api/clients/microsoft_client.py

```
@@ -76,300 +77,352 @@ async def authenticate(self) ->bool:
+                },
+            }
+            return CollectionResult(
+                evidence_type="user_directory",
+                source_system="microsoft_365",
+                resource_id="users",
+                resource_name="Azure AD Users",
+                data=evidence_data,
+                quality=quality_score,
+                compliance_controls=["CC6.1", "CC6.2", "CC6.7"],
+                collected_at=datetime.now(timezone.utc),
+            )
```

Pylint (3.3.8)

[error] Unexpected keyword argument 'evidence_type' in constructor call

(E1123)